tests/scripts/minikube.sh

```diff
@@ -120,7 +120,7 @@ function copy_images() {
 }
 
 # configure minikube
-KUBE_VERSION=${KUBE_VERSION:-"v1.14.1"}
+KUBE_VERSION=${KUBE_VERSION:-"v1.15.1"}
```

Comment:
AFAIK minikube always picks the latest Kubernetes version, why don't we let it do so? Do people override this often?

Reply:
Good point, just stuck in my old ways where we had a need at some point to run a specific version because of a minikube bug.